### DISCUSSION TITLE
It's not clear how to access assigned variables

### DISCUSSION THREAD

**Discussion by ghost** [postmanlabs/postman-flows#137] [Q&A]
I assign variable however not clear how can I access it in the next request. Assigning for example the variable id however {{id}} is not replaced with the actual value if found in the request body in the collection for the next request.

**Comment by samay-v** [comment]
You can reference the variables created in a flow the sae way you reference other variables in a request by using the {{<variable name>}} syntax. 
Know that the variables set in flows only work inside of a flow.

**Comment by YingZongLee** [comment]
The assign variable can only access the next request but not all the rest requests in the flow. Is that any way to assign the environment variable which the value is from a request in a flow?
